setup.cfg

```
@@ -26,31 +27,25 @@ zip_safe = False
 include_package_data = True
 packages = find:
 install_requires =
-    six
     colorama;platform_system=="Windows"
+python_requires = >= 2.7, != 3.0.*, != 3.1.*, != 3.2.*, != 3.3.*
```

You could probably simplify into:
```suggestion
python_requires = >= 3.4
```